packages/ui/components/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -177,6 +177,8 @@ export class LionProgressIndicator extends LocalizeMixin(LitElement) {
           this.value = this.max;
         }
       }
+    } else {
+      throw new Error('min greater than max');
```

Comment:
```suggestion
      throw new Error(`progress-indicator min value (${this.min}) should always be smaller than max value (${this.max})`);
```

Reply:
it("throws an error when min is greater than max", async () => {

      let errorCaught = false;
     try{ 
      const el = await fixture(
        html`<lion-progress-indicator min="100" max="90" aria-label="foo"></lion-progress-indicator> `,
      );
      // await el.updateComplete;
    }catch(error){ 
        errorCaught = true
        expect(error).to.be.an.instanceOf(Error);
        expect(error.message).to.equal("progress-indicator min value (100) should always be smaller than max value (90)")
      }

      console.log("error", errorCaught)
      expect(errorCaught).to.be.true
    })
    
    
    I added this test but this is not able to catch the error, seems like updated does not get called, this works fine on my setup which I have done with lit elements and I'm using lion components as base.
    
    Can you tell me what I am doing wrong

Reply:
@jorenbroekema I have committed the changes, was not able to add test cases to catch this as an error, please have a look :)

Reply:
Ah thanks for the ping, I'll take a look as soon as I can :)